core/services/chainlink/application.go

```diff
@@ -60,9 +56,9 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/plugins"
 )
 
-//go:generate mockery --quiet --name Application --output ../../internal/mocks/ --case=underscore
-
 // Application implements the common functions used in the core node.
+
```

Comment:
```suggestion
```

Reply:
As long as it's a connected block of comments :shrug: 

I don't see any change here though.